cmake/BipedalLocomotionFrameworkFindDependencies.cmake

```diff
@@ -150,6 +150,12 @@ checkandset_dependency(cppad)
 find_package(manif QUIET)
 checkandset_dependency(manif)
 
+find_package(pybind11 CONFIG)
+checkandset_dependency(pybind11)
+
+find_package(Python3 3.6 COMPONENTS Interpreter Development QUIET)
+checkandset_dependency(Python3)
```

Comment:
This probably has to be
```
 checkandset_dependency(Python3 MINIMUM_VERSION 3.6 COMPONENTS Interpreter Development )
```
but please check it. We have not been very precise in this before, nonetheless it may be worth changing.

Reply:
Done in 41c4270. Is the `find_package` still required? Logic in the CMake macro is quite convoluted to get it on the fly :)

Reply:
Yep it's still required. `checkandset_dependency` checks  only if `Python3_FOUND` exists 
https://github.com/dic-iit/bipedal-locomotion-framework/blob/be1e0a2ea0077e3ed745ec6891aac3e71d35824a/cmake/BipedalLocomotionFrameworkFindDependencies.cmake#L44

and `Python3_FOUND` is defined by `find_package(.)`

Reply:
Thanks @GiulioRomualdi for the confirmation!